src/Backend/Modules/Location/Domain/Location.php

```diff
@@ -0,0 +1,295 @@
+<?php
+
+namespace Backend\Modules\Location\Domain;
+
+use Common\Locale;
+use Backend\Core\Language\Locale as BackendLocale;
+use Frontend\Core\Language\Locale as FrontendLocale;
+use DateTime;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Table(name="location")
+ * @ORM\Entity
+ * @ORM\HasLifecycleCallbacks
+ */
+class Location
+{
+    /**
+     * @var int
+     *
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @var Locale
+     *
+     * @ORM\Column(type="locale", name="language")
+     */
+    private $locale;
+
+    /**
+     * @TODO: map this to some sort of Extra entity
```

Comment:
Dit snap ik niet.

Reply:
Misschien uitleggen in comments wat deze properties is/doet

Reply:
Well the 'modules_extras' table contains some information that could be transformed to an entity. If that is transformed we can easily map the entities together instead of using an id.

Reply:
k, so this could be changed if we switch to all Entities throughout-fork. In that case 👍